android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -228,6 +228,49 @@ public static void restoreBlocked(Context context) {
         context).getStringSet("blocked", new HashSet<String>()));
   }
 
+  /**
+   * Checks to see if name matches defined wifi direct structure.
+   * @param context to access shared preferences
+   */
+  public static boolean isWifiDirectName(String name) {
```

Comment:
Give this a different name.  You are *always* testing wifidirect names here, so call it something like isWifiDirectNameForPhysicalWeb (maybe you can come up with something better)